telegram/message.py

```diff
@@ -978,33 +978,49 @@ def _parse_html(message_text, entities, urled=False):
         html_text = ''
         last_offset = 0
 
-        for entity, text in sorted(entities.items(), key=(lambda item: item[0].offset)):
-            text = escape(text)
```

Comment:
Untill we know how telegram handles nested entities, I'm not sure if we want to change this just yet.

Reply:
I could reverse the corresponding commit and put it in a new branch, so we can get back to it when API 4.5 is released. Shall I?

Reply:
yes, that seems appropriate. It's good code if they handle it like this.

Reply:
Done :) See #1508